cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -84,6 +84,11 @@
         .u-link__no-border();
     }
 
+    &_print {
+        padding-left: unit( 13px / @base-font-size-px, em );
```

Comment:
Good call. In e910f76 I redid that line to mirror what's in L70:

```
unit( (@grid_gutter-width / 2) / @base-font-size-px - 0.25em, em );
```

Just without the 0.25em adjustment, since that's not needed here. So now it's:

```css
&_print {
    padding-left: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
    border-left: 1px solid @black;
}
```

Since that calculation is already in that LESS file, I didn't add a comment, but I can if it would be helpful.